src/card/card-header.jsx

```diff
@@ -109,12 +109,18 @@ const CardHeader = React.createClass({
       avatar = <Avatar src={this.props.avatar} style={styles.avatar} />;
     }
 
+    const {
+      title,
+      subtitle,
+      ...attrs,
```

Comment:
I hear you, though I think it's more important to keep it consistent for now.

If we want to change the name, we could possibly implement a linter for it and change it across the codebase. @callemall/material-ui thoughts?